csm/core/controllers/storage_capacity.py

```diff
@@ -23,8 +23,8 @@
 
 CAPACITY_SERVICE_ERROR = 0x3010
 
-@CsmView._app_routes.view("/api/v1/capacity")
-@CsmView._app_routes.view("/api/v2/capacity")
+# @CsmView._app_routes.view("/api/v1/capacity")
```

Comment:
Add a #TODO --> Commenting for now will re-visit and enable once CEPH capacity work is done

Reply:
updated